src/utils/layer-utils/layer-utils.js

```diff
@@ -48,24 +48,24 @@ export function findDefaultLayer(dataset, layerClasses) {
 /**
  * calculate layer data based on layer type, col Config,
  * return updated layer if colorDomain, dataMap has changed
- * @param {object} layer
- * @param {object} state
- * @param {object} oldLayerData
- * @param {object} opt
- * @returns {object} {layerData: {}, layer: {} || undefined}
+ * @param {Object} layer
+ * @param {Object} state
+ * @param {Object} oldLayerData
+ * @param {Object} opt
+ * @returns {Object} {layerData: {}, layer: {} || undefined}
```

Comment:
i noticed you are using two different annotation styles to document object types:
1. {Object} {layerData: {}, layer: {} || undefined}
2 {{layerData: {}, layer: {} || undefined}}

We should pick one and be consistent